docs/source/configuration.rst

```diff
@@ -49,15 +49,15 @@ PA0   ADC12_IN0  Input   Battery
 PA1   N/A        N/A     N/A
 PA2   ADC12_IN2  Input   Gyroscope Vref
 PA3   ADC12_IN3  Input   Gyroscope Vout
-PA4   ADC12_IN4  Input   Side left phototransistor
+PA4   ADC12_IN4  Input   Front right phototransistor
 PA5   ADC12_IN5  Input   Side right phototransistor
-PA6   ADC12_IN6  Input   Front left phototransistor
-PA7   ADC12_IN7  Input   Front right phototransistor
-PA8   GPIO       Output  Infrared emitter switch
-PA9   TIM1_CH2   Output  Speaker PWM
-PA10  GPIO       Input   Left button
-PA11  GPIO       Input   Right button
-PA12  N/A        N/A     N/A
+PA6   ADC12_IN6  Input   Side left phototransistor
+PA7   ADC12_IN7  Input   Front left phototransistor
+PA8   GPIO       Output  Left front emitter switch
```

Comment:
I would write or Front left or Left front on both, and the same for side and right cases.

Reply:
Yeah... you are right. :blush: